src/v2/components/Validators/Table/index.jsx

```diff
@@ -38,14 +38,14 @@ const ValidatorsTable = ({separate}: {separate: boolean}) => {
             <div>{identity.name || nodePubkey}</div>
           </Link>
         </TableCell>
-        <TableCell>{stake} Lamports</TableCell>
-        <TableCell>{commission}</TableCell>
-        <TableCell>{uptime}%</TableCell>
+        <TableCell>{(stake && (stake + ' Lamports')) || 'N/A'}</TableCell>
```

Comment:
would love to have a "defaultValue" implementation or something like that, feedback welcome